src/test/java/org/sagebionetworks/bridge/exporter/integration/Exporter3Test.java

```diff
@@ -512,6 +544,343 @@ public void uploadWithScheduleContext() throws Exception {
         testUpload(UPLOAD_CONTENT, false, userMetadata, expectedMetadata);
     }
 
+    @Test
+    public void demographics() throws IOException, SynapseException, InterruptedException, TimeoutException {
+        user.getClient(ForConsentedUsersApi.class)
+                .changeSharingScope(new SharingScopeForm().scope(SharingScope.ALL_QUALIFIED_RESEARCHERS)).execute();
+        TestUser researcher = TestUserHelper.createAndSignInUser(Exporter3Test.class, false, Role.RESEARCHER);
```

Comment:
Move creating the researcher account to beforeClass(), and then delete it in afterClass(). Otherwise, this will cause test accounts to pile up.